connector/src/main/java/org/jboss/as/connector/services/workmanager/DistributedWorkManagerService.java

```diff
@@ -67,13 +66,13 @@ public void start(StartContext context) throws StartException {
 
         this.value.setTransport(transport);
 
-        BlockingExecutor longRunning = (BlockingExecutor) executorLong.getOptionalValue();
+        Executor longRunning = executorLong.getOptionalValue();
         if (longRunning != null) {
             this.value.setLongRunningThreadPool(longRunning);
```

Comment:
@tadamski Is there a reason why this one isn't StatisticsExecutorImpl?

Reply:
@tadamski The github ui doesn't show the context for my question, but in case it wasn't clear: I wasn't wondering about the use of BlockingExecutor here. It's about how all the other lines in the PR where the executor is used it's wrapper in StatisticsExecutorImpl, but this one doesn't wrap it.  Perhaps it was missed?

Reply:
@bstansberry sorry, I misunderstood the question. Looking at the code I don't see any reason why long running pool should be treated differently from short term one. I think this is an omission and longRunning executor should be wrapper as well (also in non-distributed WorkManagerService).

Reply:
@tadamski I filed https://issues.redhat.com/browse/WFLY-20597 for this and will add a commmit to this PR for it.

Reply:
Done!